server/src/main/java/org/candlepin/controller/SuspendModeTransitioner.java

```diff
@@ -158,7 +125,7 @@ public void run() {
         finally {
             /**
              * The Transitioner is periodic task. But we want to have flexibility of setting
-             * different delays between executions. As the amount of failed atttempts to connect
+             * different delays between executions. As the amount of failed attempts to connect
              * rise, we want to also increase the delay. Scheduling in finally seems to be the
              * easiest way to achieve that.
              */
```

Comment:
This comment is now invalid and should be corrected. We are no longer increasing the delay.